### PR TITLE
Apply grammar suggestions from the IntelliJ linter

### PR DESCRIPTION
**Pull request by rhowe**
###### Problem:
IntelliJ's linter identified a few grammatical gremlins

###### Solution:
Apply most of them

### CODE REVIEW

File: dropwizard-configuration/src/test/java/io/dropwizard/configuration/LevenshteinComparatorTest.java [hunk 1]
```diff
@@ -11,7 +11,7 @@ class LevenshteinComparatorTest {
     private final ConfigurationParsingException.Builder.LevenshteinComparator c = new ConfigurationParsingException.Builder.LevenshteinComparator("base");
 
     /**
-     * An "java.lang.IllegalArgumentException: Comparison method violates its general contract!"
+     * A "java.lang.IllegalArgumentException: Comparison method violates its general contract!"
```
Comment (georgespalding) [inline dropwizard-configuration/src/test/java/io/dropwizard/configuration/LevenshteinComparatorTest.java]: :thought_balloon: Reading this, I "silence" the `java.lang.` part, which leads to me wanting to say "An `IllegalArgumentException` ...".

File: dropwizard-example/README.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ Included with this application is an example of the optional DB API module. The
 the features available in [Hibernate](http://hibernate.org/), along with demonstrating how these are used from within
 Dropwizard.
 
-This database example is comprised of the following classes:
+This database example comprises of the following classes:
```
Comment (georgespalding) [inline dropwizard-example/README.md]: I'm fully aware that "comprises of" is a mine field (See [npr clip](https://www.npr.org/2015/03/12/392568604/dont-you-dare-use-comprised-of-on-wikipedia-one-editor-will-take-it-out)), so please correct me if I'm wrong, but is this more correct?

```suggestion
This database example comprises the following classes:
```

**Review by georgespalding — COMMENTED** [review]
Overall, an improvement

File: dropwizard-example/README.md (hunk 2 above)
Comment (rhowe) [reply dropwizard-example/README.md]: Yes, I think that reads better. Amended.

File: dropwizard-configuration/src/test/java/io/dropwizard/configuration/LevenshteinComparatorTest.java (hunk 1 above)
Comment (rhowe) [reply dropwizard-configuration/src/test/java/io/dropwizard/configuration/LevenshteinComparatorTest.java]: I've amended with

```
An {@link IllegalArgumentException} "Comparison ..."
```

which is probably even better?

**Review by georgespalding — APPROVED** [review]
LGTM. I'm not sure what the best practice is for this repo, the PR says "out of date with base branch" so perhaps rebase onto `release/4.0.x`?

**Review by joschi — APPROVED** [review]
@rhowe Thanks! :heart:

**Pull request merged**